apps/meteor/client/views/room/contextualBar/Info/RoomInfo/ABAC/RoomInfoABACSection.tsx

```diff
@@ -0,0 +1,62 @@
+import type { IRoom } from '@rocket.chat/core-typings';
+import { Box, Divider, Tag } from '@rocket.chat/fuselage';
+import { useSetting } from '@rocket.chat/ui-contexts';
+import { useTranslation } from 'react-i18next';
+
+import { InfoPanelField, InfoPanelLabel } from '../../../../../../components/InfoPanel';
+import { RoomIcon } from '../../../../../../components/RoomIcon';
+
+// TODO: Remove type union when ABAC is implemented
+type RoomInfoABACSectionProps = {
+	room: IRoom & {
+		abacAttributes: {
+			name: string;
+			values: string[];
+		}[];
+	};
+};
+
+const RoomInfoABACSection = ({ room }: RoomInfoABACSectionProps) => {
+	const { t } = useTranslation();
+
+	const abacEnabled = useSetting('ABAC_Enabled');
+	const showAttributesInRoom = useSetting('ABAC_ShowAttributesInRooms');
+
+	if (!abacEnabled || !showAttributesInRoom || !room.abacAttributes || room.abacAttributes.length === 0) {
+		return null;
+	}
+
+	return (
+		<>
+			<Divider mb={32} width='full' />
+			<InfoPanelField>
+				<Box display='flex' mbe={16}>
+					<Tag medium>
+						<Box display='flex' alignItems='center'>
+							<RoomIcon room={room} />
+							<Box mis={2}>{t('ABAC_Managed')}</Box>
+						</Box>
+					</Tag>
+				</Box>
```

Comment:
Having the title of this section be a tag, when the section itself lists many tags seem a bit confusing. Is it intentional?